src/domain/common/profile/profile.resolver.fields.ts

```diff
@@ -76,26 +76,44 @@ export class ProfileResolverFields {
   @UseGuards(GraphqlGuard)
   @ResolveField('tagset', () => ITagset, {
     nullable: true,
-    description: 'The default tagset.',
+    description: 'The default or named tagset.',
   })
   @Profiling.api
   async tagset(
     @Parent() profile: IProfile,
+    @Args('tagsetName', {
+      type: () => TagsetReservedName,
+      nullable: true,
+    })
+    tagsetName: TagsetReservedName,
     @Loader(ProfileTagsetsLoaderCreator)
     loader: ILoader<ITagset[]>
   ): Promise<ITagset> {
     const tagsets = await loader.load(profile.id);
-    const defaultTagset = tagsets.find(
-      t =>
-        t.type === TagsetType.FREEFORM && t.name === TagsetReservedName.DEFAULT
-    );
-    if (!defaultTagset) {
+    if (!tagsetName) {
+      const defaultTagset = tagsets.find(
+        t =>
+          t.type === TagsetType.FREEFORM &&
```

Comment:
I don't see why we have to make sure here that the default tagset is of type freeform... I guess there's not going to be repeated names in the tagsets... But I see the code was already there

Reply:
The code was there but I think we also can have tagsets with same name.